examples/olm/catalogsource.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: security-profiles-operator
+  # namespace: openshift-marketplace on OCP
+  namespace: olm
+spec:
+  sourceType: grpc
+  image: quay.io/jhrozek/security-profiles-operator-catalog:v0.4.2-dev
```

Comment:
Should this image change?

Reply:
I changed the image to at least include the quay.io SPO project namespace. But I didn't push anything there yet, it's something we should do during a release. Do you have some release automation? If yes, I can add commands to push the bundles and the catalog there...

Reply:
Hm, no we build the image continuously from `main` and push it to staging in https://github.com/kubernetes-sigs/security-profiles-operator/blob/main/hack/image-cross.sh

We could do the same for the catalog image :)